python/python/bystro/prs/tests/test_preprocess_for_prs.py

```diff
@@ -171,38 +187,6 @@ def test_filter_scores_by_p_value(mock_processed_scores_df: pd.DataFrame):
     ), "Filtered scores should contain expected SNP(s)."
 
 
```

Comment:
Planning on adding back a new test for this function in future once the dust has settled on optimizing this code, for now all the changes made it really difficult to get the mock files to work for this test so I opted to take it out while changes are still being made. Tested that all this code works on a small dataset though so the only problems are related to mocking it correctly.